src/layers/core/point-density-hexagon-layer/point-density-hexagon-layer.js

```diff
@@ -24,20 +24,25 @@ import {log} from '../../../lib/utils';
 
 import {ordinalScale, linearScale} from '../../../utils/scale-utils';
 import {defaultColorRange} from '../../../utils/color-utils';
+import {getSortedCounts, getCountRangeFromPercentile} from '../../../utils/aggregation-utils';
 import {pointToHexbin} from './hexagon-aggregator';
 
 const defaultRadius = 1000;
 const defaultCoverage = 1;
 const defaultElevationRange = [0, 1000];
 const defaultElevationScale = 1;
 const defaultAggregator = pointToHexbin;
+const defaultLowerPercentile = 0;
```

Comment:
Does defining these defaults separately here instead of adding them directly in the `defaultProps` object improve readability? I think it might be cleaner to just drop these.